data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"Is this it? Is the game over now that humanity's at peace again?"`
```

Comment:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been fully implemented yet."`
```
See my reasoning and that of EjoThims above for why I chose this format.

Reply:
I'd be happy with this.